webui/react/src/shared/utils/routes.ts

```diff
@@ -37,17 +30,21 @@ export const isMouseEvent = (ev: AnyMouseEvent | React.KeyboardEvent): ev is Any
 export const isNewTabClickEvent = (event: AnyMouseEvent): boolean => {
   return event.button === 1 || event.metaKey || event.ctrlKey;
 };
-// Returns the address to the server hosting react assets
-// excluding the path to the subdirectory if any.
-export const reactHostAddress = (): string => {
-  return `${window.location.protocol}//${window.location.host}`;
-};
+
 export const ensureAbsolutePath = (url: string): string => (isAbsolutePath(url) ? url : '/' + url);
 export const filterOutLoginLocation = (location: {
   pathname: string;
 }): { pathname: string } | undefined => {
   return location.pathname.includes('login') ? undefined : clone(location);
 };
+
+export const isFullPath = (url: string): boolean => {
```

Comment:
do we need to dupe this function here and in `routes/utils`?

Reply:
originally I was worried about which files imported what

gave this another round of cleanup, put `isFullPath` back in its original place